plugins/modules/cli_command.py

```diff
@@ -110,6 +110,18 @@
     - y
     - <password>
     - y
+
+- name: Simple regexp match for multiple prompt, multiple answer(mandatory check for all prompts)
+   ansible.netcommon.cli_command:
+     command: reload in 5
+     check_all: true
+     prompt:
+     - Save\?
+     - confirm
+     answer:
+     - n
+     - y
```

Comment:
You have an indentation error here
```suggestion
- name: Simple regexp match for multiple prompt, multiple answer(mandatory check for all prompts)
  ansible.netcommon.cli_command:
    command: reload in 5
    check_all: true
    prompt:
    - Save\?
    - confirm
    answer:
    - n
    - y
```